src/main/java/uc/user/login/LoginOutputBoundary.java

```diff
@@ -2,7 +2,7 @@
 
 /**
  * LoginOutputBoundary provides methods for an interactor.
- * They should be implemented by LoginPresenter.
+ * They should be implemented by a presenter.
```

Comment:
I believe that this is a great change since the LoginOutputBoundary should not know the name of the presenter its talking to in order to follow clean architecture!